modules/mobile/app/services/mobile/v0/lighthouse_health/service.rb

```diff
@@ -32,7 +32,8 @@ def get_immunizations
         #
         def get_location(id)
           response = perform(:get, "Location/#{id}", nil, headers)
-          Rails.logger.info('Mobile Lighthouse Service, Location response', response:)
+          # remove this logging later. it doesn't have any lasting value.
+          Rails.logger.info('Mobile Lighthouse Service, Location response', response_body: response.body)
```

Comment:
I've kept this for now because I was not seeing the logging but I was seeing all of the logging around it. That puzzled me. I suspect it's because the response is too much data to log, so I'm trying to log less data. Once this is confirmed, I can remove this altogether.